sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/EventHubReactorSession.java

```diff
@@ -43,6 +45,15 @@
     private static final Symbol ENABLE_RECEIVER_RUNTIME_METRIC_NAME =
         Symbol.valueOf(VENDOR + ":enable-receiver-runtime-metric");
 
+    private static final Symbol PRODUCER_EPOCH = Symbol.valueOf(
```

Comment:
I think this is used in multiple places, iirc we have a constants class we can put these in and make them package-private?

Reply:
Moved the constants to class SymbolConstants.